src/garage/_dtypes.py

```diff
@@ -669,6 +707,10 @@ class TimeStepBatch(
         step_types (numpy.ndarray): A numpy array of `StepType with shape (
             batch_size,) containing the time step types for all transitions in
             this batch.
+        episode_infos (dict): A dict of episode-level information.
+            It contains information of he entire episode， which could be
+            needed to determine the first action (e.g. in the case of
+            goal-conditioned or MTRL.)
```

Comment:
```suggestion
        episode_infos (dict): A dict of episode-level information.
            It contains information of he entire episode, which could be
            needed to determine the first action (e.g. in the case of
            goal-conditioned or MTRL it could contain the goal or task for this episode, respectively).
```